test/functional/mempool_accept.py

```diff
@@ -105,7 +105,7 @@ def run_test(self):
             locktime=node.getblockcount() + 2000,  # Can be anything
         ))['hex']
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_final)))
-        fee_expected = int(coin['amount']) - output_amount
+        fee_expected = Decimal(str(coin['amount']) )- Decimal(str(output_amount))
```

Comment:
Odd spacing. Try to adapt the style around you (i.e. spaces before and after `-`, no space between `)` and `)`, ...

Reply:
Thank you @kallewoof. I'll be careful about styling from now on.